packages/ui/src/components/controls/Controls.module.scss

```diff
@@ -286,3 +286,36 @@
     }
   }
 }
+
+.slider {
+  display: flex;
+  justify-content: center;
+  height: 120px;
+  width: 24px;
+  cursor: pointer;
+}
+
+.sliderTrack {
+  position: relative;
+  width: 4px;
+  border-radius: var(--radius);
+  background-color: var(--background-color-dark);
+}
+
+.sliderProgress {
+  position: absolute;
+  bottom: 0;
+  width: 4px;
+  border-radius: var(--radius);
+  background-color: var(--theme);
+}
+
+.sliderThumb {
+  position: absolute;
+  left: -4px;
+  bottom: calc(100% - 2px);
+  width: 12px;
+  height: 4px;
+  border-radius: var(--radius);
+  background-color: var(--theme);
+}
```

Comment:
I adjusted the styles to match the mockup.
Mainly the width of the track should be `2px` and the handle should be an `8px` circle.
I'd also get rid of the `absolute` positioning with `calc` on the thumb. A child element will be naturally placed in the top left corner of its parent so simply nudging its position a bit to the top left will place it in the correct place:
```suggestion
.slider {
  display: flex;
  justify-content: center;
  height: 120px;
  width: 24px;
  cursor: pointer;
}

.sliderTrack {
  display: flex;
  align-items: end;
  position: relative;
  width: 2px;
  border-radius: var(--radius);
  background-color: var(--background-color-dark);
}

.sliderProgress {
  width: 2px;
  border-radius: var(--radius);
  background-color: var(--theme);
}

.sliderThumb {
  width: 8px;
  height: 8px;
  border-radius: 4px;
  margin-left: -3px;
  margin-top: -4px;
  background-color: var(--theme);
}
```